src/jit/compiler.hpp

```diff
@@ -1129,6 +1129,38 @@ inline GenTreeCall* Compiler::gtNewHelperCallNode(unsigned helper, var_types typ
     return result;
 }
 
+//------------------------------------------------------------------------------
+// gtNewReadyToRunHelperCallNode : Helper to create a call helper node ReadyToRun
+//
+// Arguments:
+//    helper    - Call helper. Has to have a valid R2R mapping (see:readytorunhelpers.h)
+//    type      - Type of the node
+//    args      - Call args
+//
+// Return Value:
+//    New CT_HELPER node
+#ifdef FEATURE_READYTORUN_COMPILER
+inline GenTreeCall* Compiler::gtNewReadyToRunHelperCallNode(unsigned helper, var_types type, GenTreeArgList* args)
```

Comment:
I do not understand why this is necessary. This is a call to a regular helper.

Reply:
Removed